etc/metrics/org.osg.local.network-monitoring-local.conf

```diff
@@ -1,13 +1,10 @@
 [org.osg.local.network-monitoring-local]
 cron-interval = */30 * * * *
 [org.osg.local.network-monitoring-local args]
-#super_mesh = http://myosg.grid.iu.edu/pfmesh/all
-#super_mesh = http://myosg.grid.iu.edu/pfmesh/all?new
-#mesh0 =  http://myosg.grid.iu.edu/psmesh/json/name/wlcg-latency-all?new
-#mesh0 = http://myosg.grid.iu.edu/psmesh/json/name/osg-goc?new
-#mesh0 = http://myosg.grid.iu.edu/pfmesh/json/name/us-atlas
-#mesh1 = http://myosg.grid.iu.edu/pfmesh/json/name/us-cms
-mesh0 = http://meshconfig-itb.grid.iu.edu/pub/config/ps-testbed
+#super_mesh = https://meshconfig.opensciencegrid.org/pub/config
+#mesh0 = https://meshconfig.grid.iu.edu/pub/config/us-atlas
+#mesh1 = ttps://meshconfig.grid.iu.edu/pub/config/us-cms
```

Comment:
Why are the atlas/cms meshconfigs still using `grid.iu.edu`? Also, there looks like an `https` typo in the cms line

Reply:
@brianhlin  Opps some typo. But it is quite some work to separate the commit . Is that really important?

Reply:
It shouldn't be that bad if you haven't made any other commits since:

```
$ git checkout master
$ git reset fe344c6
$ git add libexec/probes/worker-scripts/uploader/esmonduploader_beforeSoichiPatch.py
$ git commit -m "Drop unused patch"
$ git commit -am 'replace grid.iu.edu references with opensciencegrid.org (SOFTWARE-3209)'
$ git push -f <your github remote> master
```